synadm/api.py

```diff
@@ -600,6 +600,7 @@ def user_modify(self, user_id, password, display_name, threepid,
 
         Threepid should be passed as a tuple in a tuple
         """
+        # TODO: deprecate
```

Comment:
I agree we should deprecate `synadm user modify` within this PR, or actually I'd like to achieve this: As soon as everything from `user modify` has its own command _or_ the user should be warned - best with a literal log.warning.

But I'm not sure anymore wether we'd like to keep some parts of `user modify` as-is and not invent new commands. Can't recall what was the outcome of that discussion, please help me remember! :-)

If we _would_ keep parts of `modify` as-is and do not plan to change, then probably it would be better that exactly when these options are used, a log.warning is spit out. Not sure though what's best....